### ISSUE TITLE
Unable to modify/view certain taks

### ISSUE THREAD

**Issue by kmjayadeep** [vit-project/vit#348]
**Describe the bug**

I've encountered a weird bug with v.2.3.2. I'm unable to edit or open some of the tasks shown in the list. They are visible, and there is nothing special about them, but when i use arrow keys to select this task and try to open using ENTER, nothing happens. None of the keyboard shortcuts work either.
Currently i have ~20 tasks in the list. And 4 of them have this issue.

**To Reproduce**

I have no clue how to reproduce this. If I add 10 new tasks, maybe 2 of them will have this issue. the affected ones will be forever in this state until i manually delete them using task cli.

Here is one such task, 

```
❯ task 20 

Name          Value                               
ID            20                                  
Description   psuite auth
Status        Pending                             
Project       homelab
Entered       2023-11-18 09:48:14 (4w)            
Last modified 2023-11-18 09:55:33 (4w)
Virtual tags  PENDING PROJECT READY UDA UNBLOCKED 
UUID          5a19c75b-64b8-4cf6-b800-e829a2d58c93
Urgency       1.159                               
folder        someday

    project      1 *    1 =      1
    age      0.079 *    2 =  0.159
                            ------
                             1.159

Date                Modification                          
2023-11-18 09:55:33 Project set to 'homelab'.             
                    Folder changed from 'in' to 'someday'.
```
is there something special about this task which prevents it from being edited in VIT?
This is a fresh installation, and config.ini is empty

**Expected behavior**

I can modify/view/delete any tasks in vit

**Test case**
If reproducing your issue requires any TaskWarrior setup, please produce a [test case](https://github.com/vit-project/vit/blob/2.x/TEST-CASE.md) script.

**Comment by thehunmonkgroup** [comment]
Unfortunately I cannot fix bugs that I cannot reproduce.

There is nothing about the task that would indicate an issue, and I've never experienced this issue before.

If you can use https://github.com/vit-project/vit/blob/2.x/scripts/generate-dummy-install.sh to create a *reproducible* bug, then I can investigate.

The only other suggestion I have is a completely fresh install on another machine, as a way to determine if it's something in your environment that is causing an issue, such as overriding key press events.

**Comment by kmjayadeep** [comment]
Thanks for pointing to the script. That helped. I was able to find the issue by starting with a fresh setup using the script and copying my config&data incrementally.

It looks like my Syncthing somehow messed up syncing the files. Certain tasks ended up in both completed.data and pending.data files. Task cli was able to manage it without any issues. I assume it doesn't look up the uuid in `completed.data` unless specifically asked to. 

It is easy to reproduce the issue. Just run the dummy install script and copy `pending.data` to `completed.data`. Open VIT and you won't be able to interact with the tasks anymore. Running task command will work fine.

Anyway I was able to fix mine by deleting the problematic entries from completed.data manually. I wouldn't consider this a "bug" in VIT since the data is not supposed to be that way. But maybe it is interesting for you to check the logic on why it happened though. I'll let you decide and close the issue..

Thanks!
